3 - Assignment/240326 - Rust Basic + Roguelike Game, Week 1/nested-arrays/src/lib.rs

```diff
@@ -1,5 +1,12 @@
 pub fn transpose(matrix: [[i32; 3]; 3]) -> [[i32; 3]; 3] {
-    todo!("Implement this")
+    let mut transposed = [[0; 3]; 3];
+    for i in 0..3 {
+        for j in 0..3 {
+            transposed[i][j] = matrix[j][i];
+        }
+    }
+
+    return transposed;
```

Comment:
`return`을 생략할 수 있습니다.
```suggestion
    transposed
```